volto/razzle.config.js

```diff
@@ -61,17 +61,15 @@ module.exports.modifyWebpackConfig = ({
 
     res.plugins = [
       ...res.plugins,
-      new CopyPlugin({
-        patterns: [
-          {
-            from: path.join(
-              __dirname,
-              '/node_modules/govuk-frontend/govuk/assets',
-            ),
-            to: path.join(__dirname, '/public/assets'),
-          },
-        ],
-      }),
+      new CopyPlugin([
+        {
+          from: path.join(
+            __dirname,
+            '/node_modules/govuk-frontend/govuk/assets',
+          ),
+          to: path.join(__dirname, `/${dev ? '' : 'build/'}public/assets`),
```

Comment:
Hrm, this is still going to pollute our `public` directory with misc assets every time we run a dev server - git status afterwards shows:

```
On branch govuk-asset-issues
Your branch is up-to-date with 'origin/govuk-asset-issues'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	public/assets/
```

Reply:
Can't we just add `public/assets` to gitignore?

Reply:
Hrm, it's just that the idea of the public dir is for us to manually include any totally static assets, so in principle I don't like ignoring a dir in there.

But it's a bit philosophical; gitignore gets this over the line and looking good for the show and tell, so yeah... let's .gitignore it and move on :+1:

Reply:
Fixed